memcache.go

```diff
@@ -21,14 +21,14 @@ import (
 // these flags match pylibmc in _pylibmcmodule.h
 const (
 	FLAG_NONE    uint32 = 0
-	FLAG_PICKLE  uint32 = 1 << 0 // note 'None' and unicoee types in python get set as pickled
+	FLAG_PICKLE  uint32 = 1 << 0
 	FLAG_INTEGER uint32 = 1 << 1
 	FLAG_LONG    uint32 = 1 << 2
 	FLAG_ZLIB    uint32 = 1 << 3
-	FLAG_BOOL    uint32 = 1 << 4 // This is a pylibmc addition
+	FLAG_BOOL    uint32 = 1 << 4
```

Comment:
can we  keep the comments, but possibly without the `unicode` typo?

Reply:
Of course 👍